lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {string} tagName - name of the parent to search for.
+   * @example this.parentbyTagName('my-app');
+   */
+  parentByTagName(tagName) {
```

Comment:
yeah, if you lowercase `tagName` at the start then the only comparison you need to do is `node.tagName.toLowerCase() === tagName` and the entire thing becomes totally case-insensitive.